### PR TITLE
Remove template, Update drop, Remove drop

### PR DESCRIPTION
**Pull request by danishtroon**
* remove template
* update drop 
* remove drop

### CODE REVIEW

**Comment by saadtroon** [comment]
Please add comments and try to create pull request of all the functionality and then ask for review.

**Comment by saadtroon** [comment]
Please resolve conflicts

File: contracts/NFTContract.cdc [hunk 1]
```diff
@@ -6,6 +6,7 @@ pub contract NFTContract: NonFungibleToken {
     pub event ContractInitialized()
     pub event Withdraw(id: UInt64, from: Address?)
     pub event Deposit(id: UInt64, to: Address?)
+    pub event NFTBorrowed(id: UInt64)
```
Comment (saadtroon) [inline contracts/NFTContract.cdc:9]: Why have added this event and not used?

File: contracts/NowWhereContract.cdc [hunk 2]
```diff
@@ -56,34 +56,23 @@ pub contract NowWhereContract {
         // templates can be updated, if sale is not started yet
         pub fun updateDrop(startDate: UFix64?, endDate: UFix64?, templates: {UInt64: AnyStruct}?){
             pre{
-                (startDate==nil) || (startDate!=nil &&  self.startDate > getCurrentBlock().timestamp && startDate! >= getCurrentBlock().timestamp): "can't update start date"
-                (endDate==nil) || (endDate!=nil && endDate! > getCurrentBlock().timestamp): "can't update end date"
-                (templates==nil) || (templates != nil && templates!.keys.length != 0 && self.startDate > getCurrentBlock().timestamp) : "can't update templates"
+                (startDate==nil) || (self.startDate > getCurrentBlock().timestamp && startDate! >= getCurrentBlock().timestamp): "can't update start date"
```
Comment (saadtroon) [inline contracts/NowWhereContract.cdc:59]: please follow proper spacing bw conditions

File: transactions/updateDrop.cdc [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import NowWhereContract from "../contracts/NowWhereContract.cdc"
 
-transaction(DropId: UInt64, StartDate: UFix64,EndDate: UFix64,template: {UInt64:AnyStruct}){
+transaction(DropId: UInt64, StartDate: UFix64?,EndDate: UFix64?,template: {UInt64:AnyStruct}?){
```
Comment (saadtroon) [inline transactions/updateDrop.cdc:3]: please follow spacing to maintain consistency

File: transactions/updateDropStatic.cdc [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
+transaction(DropId: UInt64, StartDate: UFix64?,EndDate: UFix64?){
     let adminRef: &NowWhereContract.DropAdmin
     prepare(acct: AuthAccount) {
         self.adminRef = acct.borrow<&NowWhereContract.DropAdmin>(from: NowWhereContract.DropAdminStoragePath)
         ??panic("could not borrow admin reference")
     }
     execute{
-        // let template : {UInt64:AnyStruct} = {1:"3"}
-        let template : {UInt64:AnyStruct} = {}
+       // let template : {UInt64:AnyStruct} = {3:"3"}
```
Comment (saadtroon) [inline transactions/updateDropStatic.cdc:10]: you can remove this comments. Why are you keeping it?

File: contracts/NowWhereContract.cdc (hunk 2 above)
Comment (danishtroon) [reply contracts/NowWhereContract.cdc]: @saadtroon done

**Review by saadtroon — APPROVED** [review]

**Pull request merged**